recipes/newrelic/apm/php/debian.yml

```diff
@@ -42,21 +42,38 @@ preInstall:
         # PHP not installed, no detection
         exit 1
       fi
+
       # Only PHP-FPM is supported at this time
```

Comment:
Might change this comment now as more than PHP-FPM is supported?

Reply:
Good catch @mfulb I've removed this comment, feel free to re-approve.